controllers/api/userRoutes.js

```diff
@@ -50,6 +50,7 @@ router.post('/login', async (req, res) => {
     }
     // saves the session
     req.session.save(() => {
+      req.session.user_id = dbUserData.id;
```

Comment:
setting the req.session.user_id when a user logs in (not just when a new user is created)